package.json

```diff
@@ -11,7 +11,7 @@
     "start": "concurrently --kill-others \"rollup -c -w\" \"cd .. && static-server\"",
     "build": "rollup -c",
     "test": "rollup -c && jest",
-    "lint": "eslint ./src/*.js ./test/*.js",
+    "lint": "eslint \"./src/**/*.{js,ts}\" \"./test/**/*.{js,ts}\" && tsc --noEmit",
```

Comment:
You've updated both the eslint command and added the tsc command here but the eslint config hasn't been updated to support ts files. Lets just add the tsc --noEmit command and leave the eslint command alone here.

You'll also need to add latest typescript to the package.json (npm install typescript) so the tsc command works.